Ato-visionOS/Ato-visionOS/Source/Entities/LabAtom.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  LabAtom.swift
+//  Ato-visionOS
+//
+//  Created by ellllly on 7/24/25.
+//
+
+import SwiftUI
+import RealityKit
+
+class LabAtom: Atom {
+    // MARK: - Propteries
+    
+    private let id = UUID()
+    private let color1: Color
+    private let color2: Color
```

Comment:
```suggestion
    private let diffuseColor: UIColor
    private let emissiveColor: UIColor
```
네이밍 한 눈에 알 수 있게, 위에 처럼 수정해주세요! 그리고 3D 에셋을 위한 컬러라 UIColor
로 해주세용

Reply:
@wjdalswl 마이 리뷰 감사해요! 수정했어요~ 확인부탁드려요!